src/usePopper.js

```diff
@@ -98,35 +98,32 @@ export const usePopper = (
   ]);
 
   const popperInstanceRef = React.useRef();
-  const createPopper = React.useMemo(
```

Comment:
This was only used in one `useIsomorphicLayoutEffect`, so I figured we can just inline it.